server/api/attendance/attendance.controller.js

```diff
@@ -0,0 +1,674 @@
+'use strict';
+
+var _ = require('lodash');
+var Attendance = require('./attendance.model');
+var mongoose = require('mongoose');
+var ClassYear = require('../classyear/classyear.model');
+var User = require('../user/user.model');
+var SmallGroup = require('../smallgroup/smallgroup.model');
+var config = require('../../config/environment');
+
+function isoDateToTime(isoDate){
+    var date = new Date(isoDate);
+    date.setHours(0,0,0,0);
+    return date.getTime();
+}
+
+// *******************************************************
+// Get list of attendance submissions
+// Restricted to mentors
+// router.get('/', auth.isAuthenticated(), controller.index);
```

Comment:
I don't really like having this line in the function declaration because it is subject to change and requires all developers remembering to change the function comment code if they change route names, or permissions. It also is not hard at all to have the route definitions open to at the same time which has this info in it.